Flutter/lib/NameScreen.dart

```diff
@@ -1,30 +1,39 @@
-import 'package:flutter/material.dart';
-import 'package:google_fonts/google_fonts.dart';
-import 'CreateSettle.dart';
-import 'JoinSettle.dart';
-import 'app_localizations.dart';
-
-class NameScreen extends StatelessWidget {
-  final _formKey = GlobalKey<FormState>();
+import 'import_all.dart';
 
+class NameScreen extends StatefulWidget {
+  final DarkThemeProvider themeChange;
   final bool newSession;
   final context;
-  NameScreen(this.newSession, this.context);
+  NameScreen(this.newSession, this.context, this.themeChange);
+  _NameScreenState createState() => _NameScreenState();
+}
+
+class _NameScreenState extends State<NameScreen> {
+  final _formKey = GlobalKey<FormState>();
 
   void _navigate(String name) {
-    if (newSession) {
+    if (widget.newSession) {
       Navigator.push(
-        context,
-        MaterialPageRoute(builder: (context) => CreateSettle(name)),
+        widget.context,
+        MaterialPageRoute(
+            builder: (context) => CreateSettle(name, widget.themeChange)),
       );
     } else {
       Navigator.push(
-        context,
-        MaterialPageRoute(builder: (context) => JoinSettle(name)),
+        widget.context,
+        MaterialPageRoute(
+            builder: (context) => JoinSettle(name, widget.themeChange)),
       );
     }
   }
 
+  String getImage() {
+    setState(() {});
+    return widget.themeChange.darkTheme
+        ? 'assets/background-dark.png'
+        : 'assets/background.png';
```

Comment:
Seems like something AppTheme should handle

Reply:
Resolved in commit `app-style 0181911`